csharp-advanced-topics/DeprecationInCsharp/DeprecationInCsharp/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Deprecation
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            var originalString = StringUtils.input;
```

Comment:
We can just declare a new constant string here as it will make the example clearer. The suggestion for the constant string in the testing was meant to only apply within the testing assembly where we have an input and an expected output. We are using the same input in all tests and have the same expected output in all tests, so it is natural to add them as class member constants there.